src/mantidprofiler/algorithm_tree.py

```diff
@@ -134,8 +134,14 @@ def apply_multiple_trees(trees, check, func):
 
 
 def parseLine(line):
-    res = re.search("ThreadID=([0-9]*), AlgorithmName=(.*), StartTime=([0-9]*), EndTime=([0-9]*)", line)
-    return {"thread_id": res.group(1), "name": res.group(2), "start": int(res.group(3)), "finish": int(res.group(4))}
+    res = {
+        "thread_id": re.search("ThreadID=([0-9]*)", line)[1],
+        "name": re.search("AlgorithmName=(.*)", line)[1],
```

Comment:
This regexp is too greedy. It is swallowing everything to the end of the line
![image](https://github.com/user-attachments/assets/190ad4ee-605e-47e7-9ec6-bf11e1289d7f)

Reply:
Changed to
```
re.search("AlgorithmName=([a-zA-Z]*)"
```